FlowCrypt/Controllers/Inbox/Container/InboxViewContainerController.swift

```diff
@@ -60,9 +62,16 @@ final class InboxViewContainerController: TableNodeViewController {
 
     override func viewDidLoad() {
         super.viewDidLoad()
+        fetchSendAsList()
         fetchInboxFolder()
     }
 
+    private func fetchSendAsList() {
+        Task {
+            try await sendAsService.fetchList(isForceReload: true, for: appContext.user)
```

Comment:
@tomholub @sosnovsky, In which way do we have to display send as alias dropdown? As you know there's no such default dropdown feature.
Here are 3 options.

1. Use custom dropdown library such as this [one](https://github.com/AssistoLab/DropDown)
2. Make custom dropdown library.
3. Use iOS popover (Like recipient info popup when you tap recipient)
4. Use iOS action sheet (Like when you tap 3 dots in thread detail view.

Let me know which one do you prefer.

This is android screenshot
![image](https://user-images.githubusercontent.com/77344191/173537952-3129ccaf-5915-49ce-83ef-6366cd08cdb2.png)

Reply:
I prefer option 4 - iOS action sheet, it's more familiar for iOS users and easy to implement, without need for custom elements and libraries.

Reply:
Sounds good. How about you @tomholub?

Reply:
Gmail app displays actions sheet like @sosnovsky suggested.

Reply:
Just tested gmail as well. Looked good to me.